app/scripts/streams.js

```diff
@@ -0,0 +1,121 @@
+/**
+ *  RxJS in action
+ *  Chapter #
+ *  @author Paul Daniels
+ *  @author Luis Atencio
+ */
+
+const {createStore} = Redux;
+const {ofType, createDispatcher} = DispatcherUtils;
+
+const dispatcher = createDispatcher(createStore(bankingApp, {
```

Comment:
The `dispatcher` is a middleware component that is injected between the dispatch site and the actual redux store. Essentially it allows users to create functions in the form of 

```typescript
(action$: Observable<Action>, state$: Observable<State>) => Observable<Action>
```

It allows for some trippy logic where we can intercept and perform additional logic before actions reach the store.